doc/cephadm/services/osd.rst

```diff
@@ -518,19 +523,19 @@ Create a file called (for example) ``osd_spec.yml``:
 
 This means :
 
-#. Turn any available device (ceph-volume decides what 'available' is) into an
-   OSD on all hosts that match the glob pattern '*'. (The glob pattern matches
-   against the registered hosts from `host ls`) A more detailed section on
-   host_pattern is available below.
+#. Turn any available device (``ceph-volume`` decides what ``available`` means)
+   into an OSD on all hosts that match the glob pattern '``*``'. (The glob
+   pattern matches against the registered hosts returned by the command ``host
+   ls``) A more detailed section on ``host_pattern`` is available below.
```

Comment:
I left a comment on this tracker you made https://tracker.ceph.com/issues/68812#note-1 that there is actually something we can link to here on `host_pattern`